functions-and-scope.js

```diff
@@ -48,6 +91,25 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+/*
+function averageGrade(grades)
+{
+    let totaalCijfer = 0;
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let totaalCijfer = grades[0];
```